lib/pbench/server/database/alembic.ini

```diff
@@ -26,16 +32,25 @@ script_location = alembic
 # versions/ directory
 # sourceless = false
 
-# version location specification; this defaults
+# version location specification; This defaults
```

Comment:
This is a good try, but...sadly, although this file is already highly challenged in terms of capitalization, even if it weren't, we wouldn't capitalize a non-proper noun after a semicolon.  🤷

Reply:
So in PR #3180 you can see that this is different, so I'll update this to how it looks with alembic 1.9.2 generated files.